File: translations/codenet/gpt_commented/C++2P/granite/Code/s807902500.py

s, t = input().split() # Read input strings s and t from the user
s = s + s # Concatenate string s with itself, effectively doubling it

# Check if string t can be found within the doubled string s
# If found, print "Yes", otherwise print "No"
print("Yes" if t in s else "No")

File: translations/codenet/gpt_commented/C++2C/codellama/Code/s807902500.c
#include <stdio.h> // Include the iostream library for input and output operations

int main() {
  char s[100], t[100]; // Declare two string variables, s and t
  scanf("%s %s", s, t); // Read input strings s and t from the user

  strcat(s, s); // Concatenate string s with itself, effectively doubling it

  // Check if string t can be found within the doubled string s
  // If found, print "Yes", otherwise print "No"
  printf("%s\n", strstr(s, t) ? "Yes" : "No");
}

